docs/pipeline/portal_run_id.md

```diff
@@ -0,0 +1,95 @@
+# Portal Run ID
+
+You may come across people mentioning this term. The following document and explain about its attributes, property, constraint rule and use cases that intend to solve.
+
+> In summary, the `portal_run_id` is formula-based and, systematic scheme for herding the analysis run records into Portal Pipeline database; either with data "late binding" process or automated tracking process; so that one person (or a system event) would start the analysis without the wait and, still be consistent by following standard operation procedure (SOP) and best practises.
+
+![portal_run_id.png](../model/portal_run_id.png)
+
+link: `docs/model/portal_run_id.png`
```

Comment:
Sweet! 
That image explains it well, but perhaps move it under the Construct section?

Reply:
Ok, will do.

Reply:
Yup. Let us this be for Portal generation of id scheme. 

For orcabus and uuid7, we shall discuss and study; pros/cons to see whether we can adopt to (need to) the new scheme, etc.